composer.json

```diff
@@ -1,23 +1,40 @@
 {
   "name": "spryker-eco/adyen",
+  "type": "library",
   "description": "Adyen module",
   "license": "MIT",
   "require": {
     "php": ">=7.1",
+    "spryker-eco/adyen-api": "^1.0.0",
+    "spryker-shop/checkout-page": "^1.0.0 || ^2.0.0 || ^3.0.0",
     "spryker/calculation": "^4.0.0",
     "spryker/checkout-extension": "^1.0.0",
     "spryker/kernel": "^3.0.0",
```

Comment:
It would be nice if we adjust our dependency providers to use new approach of using containers and increase version of kernel to `^3.30.0`.

Reply:
Done